model/godleyIcon.h

```diff
@@ -33,15 +33,70 @@
 namespace minsky
 {
   class GodleyTableEditor;
+  
+  // Godley variables broken out in a separate structure, as copying is non-default
+  class GodleyVars: public ItemT<GodleyVars>
```

Comment:
GodleyVars doesn't need to inherit from ItemT. Rather, use multiple inheritance in GodleyIcon.